.github/workflows/CI.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - run: npm run build
       - uses: actions/setup-python@v2
```

Comment:
Would this be a good time to update our actions to their latest versions?

- [`setup-python`](https://github.com/actions/setup-python/tree/v4) has a v4 available
- [`setup-node`](https://github.com/actions/setup-node/tree/v3) has a v3 available
- [`checkout`](https://github.com/actions/checkout/tree/v3) has a v3 available

Reply:
Done, thanks.